core/domain/question_services_test.py

```diff
@@ -974,3 +974,115 @@ def test_migrate_question_state_from_v32_to_v33(self):
         cust_args = question.question_state_data.interaction.customization_args
         self.assertEqual(cust_args['choices']['value'], '')
         self.assertEqual(cust_args['showChoicesInShuffledOrder']['value'], True)
+
+    def test_migrate_question_state_from_v33_to_v34(self):
+        feedback_html = (
+            '<p>Feedback</p><oppia-noninteractive-math ' +
+            'raw_latex-with-value="&amp;quot;+,-,-,+&amp;quot' +
+            ';"></oppia-noninteractive-math>')
```

Comment:
Move `;">` to the line above (In general try to use the 80 spaces in a line to make things more readable.)

Reply:
changed